src/components/ContentNav/Item/index.spec.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Item } from '.';
+
+describe('Item', () => {
+  it('should be active', () => {
```

Comment:
I would rephrase this to say `it('should be activated on click', () => {` to better describe what you're testing for.